src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx

```diff
@@ -1,13 +1,188 @@
-import { Meta } from "@storybook/addon-docs";
-import { ComponentName } from "../../components";
-import { ComingSoon } from "../../components/coming-soon/coming-soon";
+import {Meta} from "@storybook/addon-docs";
+import {ComponentName, Frame, UnstyledList, UnstyledListItem} from "../../components";
+import {PrinciplesAccesability} from "./principles/principlesAccesability";
+import {
+    accessibilityCheckList,
+    colorContrastImage,
+    colorContrastText,
+    hierarchyOrder,
+    imageryDecorative,
+    imageryDescriptive,
+    imageryAlt,
+    navigationFocus,
+    navigationTabs
+} from "./assets";
 
 <Meta
-  title="Foundations/Accessibility"
+    title="Foundations/Accessibility"
 />
 
 <ComponentName withFoundationBackground>
-  Accessibility
+    Accessibility
 </ComponentName>
 
-<ComingSoon />
\ No newline at end of file
+- [Intro](#intro)
+- [Accessibility principles](#accessibility-principles)
+- [Color contrast](color-contrast)
+- [Imagery](#Imagery)
+- [Hierarchy and focus](#hierarchy-and-focus)
+- [Navigation](#navigation)
+- [Assistive Technology](#assistive-technology)
+- [Feedback](#feedback)
+
+## Intro
+Web Accessibility is focused on ensuring people with disabilities can equally perceive, understand, navigate, and interact with digital content. It also means that they can contribute equally like anyone else.
+Web Accessibility is based on guidelines published by The World Wide Web Consortium (W3C) for accessible content called Web Content Accessibility Guidelines or WCAG.
+
+[WCAG Quick Reference](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize#principle1)
+
+<Tip title="Get our Accessibility checklist"><a href={accessibilityCheckList} target="_blank">Download</a> our accessibility checklist file.</Tip>
+
+## Accessibility principles
+<PrinciplesAccesability/>
+
+## Color contrast
+__Text Color Contrast__<br/>
```

Comment:
why we keep only here the bold style? why can't it be `Text Color Contrast applies to text in the page...` ?
There is no information about non-text colors on purpose?